utils/data_processing.py

```diff
@@ -0,0 +1,44 @@
+"""Process and format the mesh data for our model"""
+
+from datatypes.graph import EdgeSet
+from datatypes.graph import Graph
+
+import numpy as np
+
+
+def _triangles_to_edges(faces):
+
+    edges = np.concatenate([faces[:, 0:2], faces[:, 1:3], faces[:, [0, 2]]],
+                           axis=0)
+
+    edges = np.sort(edges, axis=1)
+    unique_edges = np.unique(edges, axis=0)
+
+    senders, receivers = unique_edges[:, 0], unique_edges[:, 1]
```

Comment:
So this means that senders are the minor index and the receiver the higher index (by the sorting above). Is that it?

Reply:
This sorting allows to quickly identify duplicated edges